pkg/minikube/problem/problem_test.go

```diff
@@ -17,10 +17,92 @@ limitations under the License.
 package problem
 
 import (
+	"bytes"
 	"fmt"
+	"strings"
 	"testing"
+
+	"k8s.io/minikube/pkg/minikube/out"
 )
 
+type buffFd struct {
+	bytes.Buffer
+	uptr uintptr
+}
+
+func (b buffFd) Fd() uintptr { return b.uptr }
+
+func TestDisplay(t *testing.T) {
+	buffErr := buffFd{}
+	out.SetErrFile(&buffErr)
+	var tests = []struct {
+		description string
+		problem     Problem
+		expected    string
+	}{
+		{
+			problem:     Problem{},
+			description: "empty",
+			expected: `
+* Error: [] <no value>
```

Comment:
"empty" feels like a "change detector" test, do you mind removing it? I consider the output of an empty problem as undefined behavior.

Context: https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html

Reply:
fixed.